maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -885,7 +885,7 @@ public void promptForDbCredentialsIfMissing(Server server) {
         //set password
         String dbPassword = promptForPasswordIfMissingWithDefault(
                 "Please specify database password (-D%s)",
-                server.getDbPassword(), "dbPassword", " ");
+                server.getDbPassword(), "dbPassword", "");
         server.setDbPassword(dbPassword);
```

Comment:
I would suggest doing: 

server.setDbPassword(dbPassword.trim()); 

in order to ensure that the password does not have any leading or trailing blank spaces which might cause a authentication issue later on.